src/autofill-utils.js

```diff
@@ -47,6 +47,26 @@ const sendAndWaitForAnswer = (msgOrFn, expectedResponse) => {
     })
 }
 
+const autofillEnabled = (processConfig) => {
+    if (!isAndroid && (isDDGApp || isApp)) {
```

Comment:
What if instead of having these platform-specific checks we would just return `true` if there is no `privacyConfig`? Seems more resilient going forward. Otherwise, with every new platform that implements this, we would need to tweak this conditional.

Reply:
Seems fair. I'll make the change.